blocks/author-list/author-list.js

```diff
@@ -1,6 +1,6 @@
 import ffetch from '../../scripts/ffetch.js';
 import {
-  readBlockConfig, loadBlocks, createOptimizedPicture, decorateIcons,
+  createOptimizedPicture, decorateIcons, loadBlocks, readBlockConfig,
 } from '../../scripts/lib-franklin.js';
```

Comment:
curious was this rearranged using the tool? Asking so I can have a proper tool on my end as well :)

Reply:
eslint does that. I configured intelliJ to auto-run `eslint --fix` on each save.